include/libtorrent/add_torrent_params.hpp

```diff
@@ -297,6 +297,10 @@ namespace libtorrent
 		// url seeds to be added to the torrent (`BEP 17`_).
 		std::vector<std::string> url_seeds;
 
+		// peers to add to the torrent, to be tried to be connected to as
+		// bittorrent peers.
+		std::vector<tcp::endpoint> peers;
```

Comment:
right now, that just uses parse_magnet_uri() as well though.. so it's still not obvious how to do that without splitting that function into a private and a public version..